dash/dash-renderer/src/actions/dependencies.js

```diff
@@ -568,10 +569,20 @@ export function validateCallbacksToLayout(state_, dispatchError) {
     function validateMap(map, cls, doState) {
         for (const id in map) {
             const idProps = map[id];
+            const fcb = flatten(values(idProps));
```

Comment:
I'm trying to guess what `fcb` stands for…

Reply:
flatten callbacks?

Reply:
aahh... thanks